src/cartesian/ticks/utils.ts

```diff
@@ -0,0 +1,59 @@
+import { CartesianTickItem } from '../../util/types';
+
+/**
+ * Given an array and a number N, return a new array which contains every nTh
+ * element of the input array. For n below 1, an empty array is returned.
+ * @param {any[]} array An input array.
+ * @param {integer} n A number
+ * @returns {any[]} The result array of the same type as the input array.
+ */
+function getEveryNth<Type>(array: Type[], n: number): Type[] {
+  if (n < 1) {
+    return [];
+  }
+  if (n === 1) {
+    return array;
+  }
+  const result = [];
+  for (let i = 0; i < array.length; i += n) {
+    result.push(array[i]);
+  }
+  return result;
+}
+
+/**
+ * Given an array of ticks, find N, the lowest possible number for which every
+ * nTH tick in the ticks array isShow == true and return the array of every nTh tick.
+ * @param {CartesianTickItem[]} ticks An array of CartesianTickItem with the
+ * information whether they can be shown without overlapping with their neighbour isShow.
+ * @returns {CartesianTickItem[]} Every nTh tick in an array.
+ */
+export function getEveryNThTick(ticks: CartesianTickItem[]) {
```

Comment:
nit nit: personally think `getEveryNthTick` looks better

Reply:
For me the name itself is a bit off. Having N in the function name suggests that it is a parameter. I do not have a good suggestion though.